charts/base-cluster/templates/weave/_helpers.tpl

```diff
@@ -0,0 +1,3 @@
+{{- define "weaveworks.password" -}}
+{{ include "common.secrets.passwords.manage" (dict "secret" "cluster-user-auth" "key" "password" "providedValues" (list "weaveworks.adminUser.password") "length" 20 "strong" false "skipB64enc" true "skipQuote" true "context" $) | bcrypt }}
+{{- end -}}
```

Comment:
This is only used once, please inline

Reply:
well this must be changed anywayse. As you also mentioned in the comment below. This won't work

Reply:
deleted